src/main/java/frc/robot/commands/RunIntakeBackwards.java

```diff
@@ -0,0 +1,55 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants;
+import frc.robot.subsystems.Intake;
+import frc.robot.util.TunableNumber;
+
+public class RunIntakeBackwards extends CommandBase {
+
+  private TunableNumber setpoint = new TunableNumber("Intake Backwards/setpoint");
+  private final Intake intake;
+
+  /**
+   * Creates a new RunIntakeBackwards.
+   * 
+   * @param Intake
+   */
+  public RunIntakeBackwards(Intake intakeSub) {
+    intake = intakeSub;
+    addRequirements(intakeSub);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    setpoint.setDefault(0);
```

Comment:
We should probably have a real default
```suggestion
    setpoint.setDefault(1);
```